stable/artifactory-ha/templates/ingress.yaml

```diff
@@ -29,7 +29,7 @@ spec:
   - host: {{ $host | quote }}
     http:
       paths:
-        - path: /
+        - path: {{ .Values.ingress.path }}
```

Comment:
Should this have been `$.Values.ingress.path` instead? I'm trying to upgrade to `0.11.14` and have ran into an issue around this templating. As I understand it, the `.` no longer points to the root context inside a range function (https://github.com/helm/helm/issues/1311#issuecomment-399577623)

Reply:
@stevie- can you verify this?

Reply:
@jaspermarcus - we are fixing it with https://github.com/jfrog/charts/pull/265.

Reply:
Ahh, Thanks, missed that. Guess tests coverage needs to be udpated as well.

Reply:
Thanks for the quick response! 👍